jni/src/main/java/com/horizen/commitmenttreenative/ScCommitmentCertPath.java

```diff
@@ -0,0 +1,83 @@
+package com.horizen.commitmenttreenative;
+
+import java.util.Optional;
+
+import com.horizen.librustsidechains.FieldElement;
+import com.horizen.librustsidechains.Library;
+
+public class ScCommitmentCertPath implements AutoCloseable {
```

Comment:
Let's add a comment maybe specifying what does this path represent (it's not exactly a classical merkle path)

Reply:
Done